data/common.yaml

```diff
@@ -2,9 +2,34 @@
 profile_metrics::cilogon_client_id: ""
 profile_metrics::cilogon_client_secret: ""
 profile_metrics::db_name: "grafana"
-profile_metrics::db_passwd: ""
+profile_metrics::db_passwd: "CHANGEME"
 profile_metrics::db_user: "grafana"
 profile_metrics::grafana_server_root_url: "https://%{facts.fqdn}"
+profile_metrics::grafana_version: "installed"
 
 profile_metrics::ssh::metrics_node_ips:
   - "127.0.0.1"
+profile_metrics::ssh::sshkey_priv: ""
+profile_metrics::ssh::sshkey_pub: ""
+profile_metrics::ssh::sshkey_type: "rsa"
+
+profile_metrics::tools::backup_and_sync_cron_hour: "4"
+profile_metrics::tools::backup_and_sync_cron_minute: "0"
+profile_metrics::tools::backup_and_sync_cron_month: "*"
+profile_metrics::tools::backup_and_sync_cron_monthday: "*"
+profile_metrics::tools::backup_and_sync_cron_weekday: "*"
```

Comment:
I dislike a hash, because that requires setting everything in the hash to override just an hour, day, etc.
And it is more complicated to understand and use for someone who might not be as much of a Puppet expert.

Reply:
I was thinking the hash would necessarily include all scheduling components (I think this is best-practice for puppet cron for the exact reason you specify).

I also think it's beneficial in that it makes the hiera easier to read due to all these related settings being indented as all controlling one aspect.

Also shortens the puppet code a bit, making it easier to read/maintain.

That being said, I'll open an enhancement request for the change to move the discussion there.